### ISSUE TITLE
JAVA_HOME is set to an invalid directory

### ISSUE THREAD

**Issue by leolux** [GoogleChromeLabs/bubblewrap#205]
I am getting the following error message when using the existing Android Tools from the Studio:
`JAVA_HOME is set to an invalid directory`

The warning is wrong becauese the JAVA_HOME is set to an valid directory. Otherwise all other IDEs and projects would not function. Which version of Java does bubblewrap expect?

**Comment by andreban** [comment]
It sounds like the path to the JDK in your config may be invalid. Can you check your configuration:
https://github.com/GoogleChromeLabs/bubblewrap/tree/master/packages/cli#updating-the-location-of-the-jdk-and--or-the-android-command-line-tools #181 will automate this in the future.

JDK requirements are documented here: https://github.com/GoogleChromeLabs/bubblewrap/tree/master/packages/cli#updating-the-location-of-the-jdk-and--or-the-android-command-line-tools

TL;DR is that the requirement comes from the Android CLI Tools. Bubblewrap doesn't use the JDK directly.

**Comment by leolux** [comment]
Ok thanks!  I guess that path to JDK 11 is causing the problem not because the path is wrong but because Android CLI Tools does not work with JDK 11.
